### ISSUE TITLE
[tune] PopulationBasedTraining and Tensorboard HPARAMS

### ISSUE THREAD

**Issue by viotemp1** [ray-project/ray#11612]
Hello,

I set in config fixed learning rate and dropout and let PopulationBasedTraining do it's mutations.
Something like this:
hyperparameter_space = {
    "lr": 1e-3,
    "dense_1": tune.quniform(32, 128, 32), 
    "count_dense_layers": tune.quniform(1, 4, 1),
    "dense_x": tune.quniform(32, 256, 32),
    "dropout_rate": 0.0,
    "tf_optimizer": tune.choice(["SGD", "Adam"]),
}
pbt_hyperparameter_space = {
    "lr": tune.loguniform(1e-4, 1e-2),
    "dropout_rate": tune.quniform(0.0, 0.3, 0.1), #
}

Algorithm is running fine, lr and dropout are tuned: config.lr=0.001346 and config.dropout_rate=0.1

Can the whole process of tuning with PopulationBasedTraining reflect also in Tensorboard?
As you can see bellow, the tensorboard show only values from hyperparameter_space.
![Screenshot 2020-10-26 at 16 49 30](https://user-images.githubusercontent.com/34300519/97187589-420e1e00-17ab-11eb-96eb-d018f7189899.png)

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
